### PR TITLE
Download temp packages within /data

### PR DESCRIPTION
**Pull request by Kimchelly**

### CODE REVIEW

File: repobuilder/job.go [hunk 1]
```diff
@@ -207,7 +207,7 @@ func (j *repoBuilderJob) setup() {
 		}
 	}
 
-	j.tmpdir, err = ioutil.TempDir("", j.ID())
+	j.tmpdir, err = ioutil.TempDir("/data", j.ID())
```
Comment (tychoish) [inline repobuilder/job.go]: make this /data/tmp

Comment (Kimchelly) [reply repobuilder/job.go]: Done, and did a sanity `MkdirAll`.

File: repobuilder/job.go [hunk 2]
```diff
@@ -207,7 +207,11 @@ func (j *repoBuilderJob) setup() {
 		}
 	}
 
-	j.tmpdir, err = ioutil.TempDir("", j.ID())
+	if err = os.MkdirAll("/data/tmp"); err != nil {
+		j.AddError(errors.Wrap(err, "could not create temp directory"))
+		return
+	}
```
Comment (tychoish) [inline repobuilder/job.go]: what if this does exist?

Comment (Kimchelly) [reply repobuilder/job.go]: If it exists already, doesn't `MkdirAll` just no-op?

Comment (tychoish) [reply repobuilder/job.go]: I think it's right but you have to put perms?

**Pull request merged**